go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
 	github.com/tendermint/go-amino v0.16.0
-	github.com/tendermint/tendermint v0.35.4
+	github.com/tendermint/tendermint v0.36.0-dev.0.20220512083648-12fed0ed5313
```

Comment:
Is there a way to keep v0.35.*?

If not, then probably this PR shouldn't go in v046

Reply:
It is 0.35.x release branch, it's just haven't released.